src/module/actor/entity.js

```diff
@@ -14,7 +14,8 @@ const removeFalsyElements = (arr) =>
 
 export default class OseActor extends Actor {
   prepareDerivedData() {
-    if (game.version.startsWith("10")) this.system.prepareDerivedData?.();
+    //if (game.version.startsWith("10")) this.system.prepareDerivedData?.();
+    this.system.prepareDerivedData?.();
```

Comment:
Did this get fixed in v10's patch cycle? v10 doesn't automatically call the data model's `prepareDerivedData`, but v11 (and later, I imagine) does.

Reply:
You may know more than me, but under v12 the `prepareDerivedData` method was not being called (the character sheet form didn't update). It appears to work without issue in v11. It's not clear to me what the change is between v11 and v12 though.

Reply:
It appears that in v12 `prepareDerivedData` is only called automatically if the new TypeDataModel classes are used.
https://github.com/foundryvtt/foundryvtt/issues/10411

Updating the classes to use TypeDataModel should resolve this and make calling prepareDerivedData unnecessary.

https://github.com/vttred/ose/blob/339b156cefd6cc17262a89f95b9d43b2024dd01c/src/ose.js#L78C1-L89C5

Reply:
@apewall Thank you! Will this end support for v10 then? Is that ok?

Reply:
I believe intention is to only support the latest stable (v12) and maintenance (v11) releases of foundryvtt. 

So it should be okay to drop v10 specific code like this.